server/src/main/java/com/networknt/server/Server.java

```diff
@@ -262,8 +269,8 @@ static private boolean bind(HttpHandler handler, int port) {
             server.start();
             System.out.println("HOST IP " + System.getenv(STATUS_HOST_IP));
         } catch (Exception e) {
-            if (!serverConfig.dynamicPort || (serverConfig.dynamicPort && serverConfig.maxPort == port + 1)) {
-                String triedPortsMessage = serverConfig.dynamicPort ? serverConfig.minPort + " to: " + (serverConfig.maxPort - 1) : port + "";
+            if (!serverConfig.dynamicPort || bindCounter > (serverConfig.maxPort - serverConfig.maxPort)) {
```

Comment:
Hi! Is this a typo? I think it should be `serverConfig.maxPort - serverConfig.minPort`

Reply:
It was my mistake. Thanks a lot for pointing it out.